lms/static/js/learner_analytics_dashboard/LearnerAnalyticsDashboard.jsx

```diff
@@ -96,9 +93,13 @@ export function LearnerAnalyticsDashboard(props) {
             </div>
           }
 
-          <h3 className="section-heading">Graded Assessments</h3>
+          <h3 className="section-heading">Graded Assignments</h3>
           <div className="graded-assessments-wrapper">
```

Comment:
Class `graded-assessments-wrapper` should be `graded-assignments-wrapper`.
Options:
1. Do nothing.
2. Add comment to fix later if needed.
{/* TODO: LEARNER-3854: If implementing Learner Analytics, rename to graded-assignments-wrapper. */}
3. Update and fix associated classes if any.

Reply:
In the interests of time, and given that this is a test, I think I will go for option 1.

Reply:
Sounds good.